### PR TITLE
feat: add operator command injection with pause/resume/abort

### LINKED ISSUE #18: #7d State Machine: operator command injection
Parent: #7 | Depends on: #7a

Accept operator override commands that take priority over autonomous navigation.

## Scope (one PR)
- OperatorCommand input (from UI or API): ABORT, PAUSE, RESUME, REDIRECT
- ABORT → immediate LANDING transition from any state
- PAUSE → hover in place, stop accepting nav decisions
- RESUME → return to previous state, re-accept nav decisions
- REDIRECT → update exploration target (stretch)
- Operator commands always override autonomous decisions
- Expose API endpoint: POST /operator-command

## Test
- PAUSE during EXPLORING, verify hover + nav rejection
- RESUME after PAUSE, verify nav re-accepted
- ABORT from any state, verify LANDING

## Session: Alpha

### PR DESCRIPTION
**Pull request by aria-DFWA**
## Summary
- `src/breacheye/state_machine.py` — adds `_paused` flag, `pause()`/`resume()`/`abort()` methods. `accepts_nav()` now gates on pause state.
- `src/breacheye/operator.py` — `OperatorCommand` model + `OperatorHandler` dispatching ABORT/PAUSE/RESUME.
- `src/breacheye/service.py` — `POST /operator-command` endpoint, FSM added to HarnessRuntime, WS topics extended with state_change/paused/resumed/abort.
- 13 async tests covering all operator actions, event publishing, idempotency, abort from every reachable state.

Closes #18.

## Test plan
- [x] `pytest tests/test_operator.py tests/test_state_machine.py -v` — 30/30 passed (no regressions)

### CODE REVIEW

File: src/breacheye/state_machine.py [hunk 1]
```diff
@@ -62,14 +62,45 @@ def __init__(self, adapter: DroneAdapter, bus: AsyncEventBus) -> None:
...
+
+    async def resume(self) -> None:
+        """Resume autonomous navigation."""
+        if not self._paused:
+            return
+        self._paused = False
+        await self._bus.publish("drone.resumed", {"timestamp": time()})
+
+    async def abort(self) -> None:
+        """Emergency abort — transition to LANDING from any non-terminal state."""
```
Comment (mcande21) [inline src/breacheye/state_machine.py:96]: **[MEDIUM] abort() from PREFLIGHT raises uncaught RuntimeError**

The LANDING guard checks `drone_state.flying`. If the drone is in PREFLIGHT (not flying), the guard raises `RuntimeError("cannot land: drone is not flying")`, not `InvalidTransition`. This propagates uncaught from `abort()`.

**Fix:** Add `FlightState.PREFLIGHT` to the early-return alongside COMPLETE. Also clear `self._paused = False` at the top of `abort()` to avoid stale state.

**Review by mcande21 — DISMISSED** [review]

**Review by mcande21 — APPROVED** [review]
LGTM. Abort skips PREFLIGHT cleanly, clears paused flag, two new edge case tests. 32/32 passing.
